### PR TITLE
Complete Kecleon Bros + Clean Kecleon/Kangaskhan

### PR DESCRIPTION
**Pull request by Kermalis**

### CODE REVIEW

File: asm/code_8098BDC.s [hunk 1]
```diff
@@ -5573,7 +5573,7 @@ _0809BC26:
 	lsrs r4, 31
 	bl sub_809B294
 	adds r0, r4, 0
-	bl CreateKangaskhanStorage
```
Comment (AnonymousRandomPerson) [inline asm/code_8098BDC.s:5576]: Per Discord conversation, let's keep the existing naming scheme here.

Comment (Kermalis) [reply asm/code_8098BDC.s]: Addressed

File: src/kecleon_bros.c [hunk 2]
```diff
@@ -0,0 +1,1777 @@
...
+u8 sub_801A37C(void);
+void sub_801A398(u8);
+void sub_801A3DC(void);
+void sub_801A430(void);
+void sub_801A4A4(void);
+void sub_801A998(void);
+s32 sub_801AE24(u32);
+
+u32 CountKecleonItems(void);
+void Kecleon_SortItems(void);
```
Comment (AnonymousRandomPerson) [inline src/kecleon_bros.c:112]: Let's swap these two functions to PascalCase too.

Comment (Kermalis) [reply src/kecleon_bros.c]: Addressed

**Review by AnonymousRandomPerson — APPROVED** [review]

**Pull request merged**